src/sql/workbench/services/errorMessage/browser/errorMessageDialog.ts

```diff
@@ -109,6 +111,14 @@ export class ErrorMessageDialog extends Modal {
 	private updateDialogBody(): void {
 		DOM.clearNode(this._body!);
 		DOM.append(this._body!, DOM.$('div.error-message')).innerText = this._message!;
+		if (this._instructionText) {
+			let childElement = DOM.$('div.error-instruction-text');
+			childElement.innerText = this._instructionText!;
+			if (this._readMoreLink) {
+				childElement.innerHTML += ` <a href="${this._readMoreLink}">Read more</a>`;
```

Comment:
1. Read More should be localized
2. You should preferably use the [Link](https://github.com/Microsoft/azuredatastudio/blob/26455e9113fb97398b2c86273af64c42d218a71a/src/vs/platform/opener/browser/link.ts) class instead of directly creating the HTML

Reply:
Link would not work in this case as in this case we want link to be appended to the same DIV element. I don't think we need to add much complexity if it's not straightforward to achieve.

Reply:
The point of using the link is that you get all accessibility, behavior and style stuff for free so that links behave and look the same across the entire application. You can just create an empty div to attach it to if you don't have an appropriate container for it already.

Reply:
I guess I'll have to update it eventually to fix build errors as it wouldn't allow me to set innerHTML :/